### ISSUE TITLE
[Catalog] Switch Control users cannot dismiss the "Settings" dialog

### ISSUE THREAD

**Issue by romoore** [material-components/material-components-ios#4743]
The Settings dialog which provides "Show Touches" and "Show Safe area inset" options does not make the "Dismiss" button accessible to Switch Control users. If they ever open the menu, they will need to quit and relaunch the app.

## Reproduction steps
1. Turn on Switch Control
1. Open the Catalog
1. Open the "..." menu in the upper-right corner of the screen
1. Choose the "Settings" option in the action sheet that appears

### Expected result
Both toggles and the "Dismiss" button can be accessed by Switch Control

### Actual result
The "Dismiss" button is inaccessible.
